src/dot.rs

```diff
@@ -188,7 +191,7 @@ impl<T> fmt::Display for Escaped<T>
 {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         if f.alternate() {
-            write!(&mut Escaper(f), "{:#}\\l", &self.0)
+            write!(&mut Escaper(f), "{:#}\n", &self.0)
```

Comment:
Is there a reason for this change?

Reply:
Yes, since backslashes are escaped now, `\\l` turns to `\\\\l` which is not a the intent. Since `\\l` turns to `\n`, I put the converted form there directly to avoid escaping `\\l`.

Reply:
ah, of course. nice

Reply:
I got a little confused there. `\\l` would turn to `\\\\l`, and in order to get `\\l` in the output, `\n` is used.